src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -290,3 +288,25 @@ describe('Font Size: 22', () => {
   // run the snapshot tests at font size 22
   testSuite()
 })
+
+describe('Drag and Drop simalation tests', () => {
+  beforeEach(removeHUD)
+
+  it('Check simulateDrag and simulateDrop effect on DOM', async () => {
+    await snapshotSimulation()
+
+    await paste(`
+      - a
+      - b
+      - c
+      - d
+    `)
+
+    await simulateDragAndDrop('[aria-label="bullet"]')
+
+    await sleep(2000)
```

Comment:
Can this delay be made any shorter?

Reply:
Sure, I reduced it 500ms, 2000ms was just for testing purpose.